README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
```

Comment:
I feel this Twitter follow link makes Cascade somehow personal; We want to make it a community-based project. So I suggest removing this badge.